wdl-lint/src/rules/preamble_formatting.rs

```diff
@@ -208,12 +202,9 @@ impl Visitor for PreambleFormattingRule {
                 .expect("should have a next token");
             if next_token.kind() != SyntaxKind::Comment {
                 // The next token must be part of the version statement
-                // and since we've already established there's a prior comment,
-                // this whitespace must be _exactly_ two newlines.
-                if s != "\r\n\r\n" && s != "\n\n" {
-                    state.add(expected_blank_line_before_version(whitespace.span()));
-                }
-                return;
+                // and we should have already returned during the above check
+                // for the version statement
+                unreachable!("next token should be a comment");
```

Comment:
Perhaps just an `assert!` instead of the conditional?

```rust
assert!(next_token.kind() == SyntaxKind::Comment, "next token should be a comment");
```